flink-yarn/src/main/java/org/apache/flink/yarn/configuration/YarnConfigOptions.java

```diff
@@ -54,6 +54,7 @@
      * (LAST), or be positioned based on their name (ORDER). DISABLED means the user-jars are
      * excluded from the system class path.
      */
+    @Deprecated
```

Comment:
Hi @knaufk, just want to mention that  [FLINK-24897](https://issues.apache.org/jira/browse/FLINK-24897) will reuse the `CLASSPATH_INCLUDE_USER_JAR` option to support using `usrlib` with user classloader in YARN application mode. As a result, it should not be deprecated and we have renamed the key of `CLASSPATH_INCLUDE_USER_JAR` from `yarn.per-job-cluster.include-user-jar` to `yarn.classpath.include-user-jar` for better naming in #18531.

Reply:
Thanks for the hint!